src/k8s/cmd/k8s/k8s_get_join_token.go

```diff
@@ -21,6 +21,10 @@ func newGetJoinTokenCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 		PreRun: chainPreRunHooks(hookRequireRoot(env)),
 		Args:   cmdutil.MaximumNArgs(env, 1),
 		Run: func(cmd *cobra.Command, args []string) {
+			if !opts.worker && len(args) == 0 {
+				cmd.PrintErrln("Error: A node name is required for control-plane nodes.")
```

Comment:
Nice! I think this is very useful. otherwise the MC error will cause a lot of confusion.